### ISSUE TITLE
Program crashing when re-initiating call.

### ISSUE THREAD

**Issue by Jelop** [Microsoft/WebRTC-universal-samples#39]
When a call is initiated from the hololens, hung up and then re-initiated the program crashes. With mixed debugging enabled I get a large amount of output. I've included from the hololens first connecting to the server to the end. But I have omitted many repeated D3D warnings and thread exit messages. One instance of the D3D warning, on the off chance that is relevant is:

```
D3D11 WARNING: ID3D11DeviceContext::Draw: Input vertex slot 0 has stride 12 which is less than the minimum stride logically expected from the current Input Layout (40 bytes). This is OK, as hardware is perfectly capable of reading overlapping data. However the developer probably did not intend to make use of this behavior.  [ EXECUTION WARNING #355: DEVICE_DRAW_VERTEX_BUFFER_STRIDE_TOO_SMALL]
D3D11 WARNING: ID3D11DeviceContext::Draw: Vertex Buffer at the input vertex slot 0 is not big enough for what the Draw*() call expects to traverse. This is OK, as reading off the end of the Buffer is defined to return 0. However the developer probably did not intend to make use of this behavior.  [ EXECUTION WARNING #356: DEVICE_DRAW_VERTEX_BUFFER_TOO_SMALL]
D3D11 WARNING: ID3D11DeviceContext::Draw: Input vertex slot 0 has stride 12 which is less than the minimum stride logically expected from the current Input Layout (40 bytes). This is OK, as hardware is perfectly capable of reading overlapping data. However the developer probably did not intend to make use of this behavior.  [ EXECUTION WARNING #355: DEVICE_DRAW_VERTEX_BUFFER_STRIDE_TOO_SMALL]
D3D11 WARNING: ID3D11DeviceContext::Draw: Vertex Buffer at the input vertex slot 0 is not big enough for what the Draw*() call expects to traverse. This is OK, as reading off the end of the Buffer is defined to return 0. However the developer probably did not intend to make use of this behavior.  [ EXECUTION WARNING #356: DEVICE_DRAW_VERTEX_BUFFER_TOO_SMALL]
D3D11 WARNING: ID3D11DeviceContext::Draw: Input vertex slot 0 has stride 12 which is less than the minimum stride logically expected from the current Input Layout (40 bytes). This is OK, as hardware is perfectly capable of reading overlapping data. However the developer probably did not intend to make use of this behavior.  [ EXECUTION WARNING #355: DEVICE_DRAW_VERTEX_BUFFER_STRIDE_TOO_SMALL]
D3D11 WARNING: ID3D11DeviceContext::Draw: Vertex Buffer at the input vertex slot 0 is not big enough for what the Draw*() call expects to traverse. This is OK, as reading off the end of the Buffer is defined to return 0. However the developer probably did not intend to make use of this behavior.  [ EXECUTION WARNING #356: DEVICE_DRAW_VERTEX_BUFFER_TOO_SMALL]
D3D11 WARNING: ID3D11DeviceContext::Draw: Input vertex slot 0 has stride 12 which is less than the minimum stride logically expected from the current Input Layout (40 bytes). This is OK, as hardware is perfectly capable of reading overlapping data. However the developer probably did not intend to make use of this behavior.  [ EXECUTION WARNING #355: DEVICE_DRAW_VERTEX_BUFFER_STRIDE_TOO_SMALL]
D3D11 WARNING: ID3D11DeviceContext::Draw: Vertex Buffer at the input vertex slot 0 is not big enough for what the Draw*() call expects to traverse. This is OK, as reading off the end of the Buffer is defined to return 0. However the developer probably did not intend to make use of this behavior.  [ EXECUTION WARNING #356: DEVICE_DRAW_VERTEX_BUFFER_TOO_SMALL]
Exception thrown at 0x77C9C7D2 in AR_Maintenance.exe: Microsoft C++ exception: _com_error at memory location 0x0FBF9AE0.
```

Partial log begins here.

```
Signaling: Headers received [i=424 data(471)]
The thread 0x5c4 has exited with code 0 (0x0).
onecoreuap\shell\windows.storage\dataaccess.cpp(2863)\windows.storage.dll!72D679B2: (caller: 72D65927) LogHr(115) tid(c50) 8007029C An assertion failure has occurred.
onecoreuap\shell\windows.storage\dataaccess.cpp(2863)\windows.storage.dll!72D679B2: (caller: 72D65927) LogHr(116) tid(c50) 8007029C An assertion failure has occurred.
The thread 0x1440 has exited with code 0 (0x0).
The thread 0xd94 has exited with code 0 (0x0).
Conductor: Creating peer connection.
Conductor: Getting user media.
The thread 0x1158 has exited with code 0 (0x0).
'AR_Maintenance.exe' (Win32): Loaded 'C:\Windows\System32\Windows.Media.MixedRealityCapture.dll'. Cannot find or open the PDB file.
The thread 0x750 has exited with code 0 (0x0).
The thread 0x868 has exited with code 0 (0x0).
VideoEffect Added
'AR_Maintenance.exe' (Win32): Loaded 'C:\Windows\System32\deviceaccess.dll'. Symbols loaded.
Conductor: Adding local media stream.
Data Channel state in initialistion: Connecting
'AR_Maintenance.exe' (Win32): Loaded 'C:\Windows\System32\Windows.Mirage.Internal.Capture.Pipeline.ProxyStub.dll'. Cannot find or open the PDB file.
Conductor: Sending offer.
'AR_Maintenance.exe' (Win32): Loaded 'C:\Windows\System32\Windows.Shell.ServiceHostBuilder.dll'. Symbols loaded.
'AR_Maintenance.exe' (Win32): Loaded 'C:\Windows\System32\Windows.Web.dll'. Symbols loaded.
'AR_Maintenance.exe' (Win32): Unloaded 'C:\Windows\System32\RTMediaFrame.dll'
Exception thrown: 'System.TimeoutException' in PeerConnectionClientCore.dll
Timed out long polling, re-trying.
Conductor: Sending ice candidate.
{"sdpMid":"audio","sdpMLineIndex":0,"candidate":"candidate:3581966237 1 udp 2122260223 172.16.100.228 58566 typ host generation 0 ufrag GQlT network-id 1 network-cost 50"}
Signaling: Headers received [i=409 data(413)]
Conductor: Sending ice candidate.
{"sdpMid":"video","sdpMLineIndex":1,"candidate":"candidate:3581966237 1 udp 2122260223 172.16.100.228 58567 typ host generation 0 ufrag GQlT network-id 1 network-cost 50"}
Signaling: Headers received [i=423 data(3772)]
Conductor: Sending ice candidate.
{"sdpMid":"data","sdpMLineIndex":2,"candidate":"candidate:3581966237 1 udp 2122260223 172.16.100.228 58568 typ host generation 0 ufrag GQlT network-id 1 network-cost 50"}
Conductor: Sending ice candidate.
{"sdpMid":"audio","sdpMLineIndex":0,"candidate":"candidate:2600700781 1 tcp 1518280447 172.16.100.228 49935 typ host tcptype passive generation 0 ufrag GQlT network-id 1 network-cost 50"}
Conductor: Received session description: {"type":"answer","sdp":"v=0\r\no=- 1758150563495385465 2 IN IP4 127.0.0.1\r\ns=-\r\nt=0 0\r\na=group:BUNDLE audio video data\r\na=msid-semantic: WMS stream_label_f22a70757163fb02\r\nm=audio 9 UDP/TLS/RTP/SAVPF 111 103 104 9 102 0 8 106 105 13 110 112 113 126\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:0wiE\r\na=ice-pwd:M5Q8jN2soyvQMce/AA5dUS1V\r\na=ice-options:trickle\r\na=fingerprint:sha-256 B0:99:60:05:EA:88:56:D2:5E:74:9C:47:76:61:27:13:A6:15:BE:67:89:BD:D8:5A:AD:63:F0:08:9E:C2:79:F2\r\na=setup:active\r\na=mid:audio\r\na=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level\r\na=sendrecv\r\na=rtcp-mux\r\na=rtpmap:111 opus/48000/2\r\na=rtcp-fb:111 transport-cc\r\na=fmtp:111 minptime=10;useinbandfec=1\r\na=rtpmap:103 ISAC/16000\r\na=rtpmap:104 ISAC/32000\r\na=rtpmap:9 G722/8000\r\na=rtpmap:102 ILBC/8000\r\na=rtpmap:0 PCMU/8000\r\na=rtpmap:8 PCMA/8000\r\na=rtpmap:106 CN/32000\r\na=rtpmap:105 CN/16000\r\na=rtpmap:13 CN/8000\r\na=rtpmap:110 telephone-event/48000\r\na=rtpmap:112 telephone-event/32000\r\na=rtpmap:113 telephone-event/16000\r\na=rtpmap:126 telephone-event/8000\r\na=ssrc:1504669606 cname:qLzAky2BD0J6pW3Z\r\na=ssrc:1504669606 msid:stream_label_f22a70757163fb02 audio_label_9267b376cd37afa6\r\na=ssrc:1504669606 mslabel:stream_label_f22a70757163fb02\r\na=ssrc:1504669606 label:audio_label_9267b376cd37afa6\r\nm=video 9 UDP/TLS/RTP/SAVPF 100 96 97 98 99 101 127 124 125\r\nc=IN IP4 0.0.0.0\r\na=rtcp:9 IN IP4 0.0.0.0\r\na=ice-ufrag:0wiE\r\na=ice-pwd:M5Q8jN2soyvQMce/AA5dUS1V\r\na=ice-options:trickle\r\na=fingerprint:sha-256 B0:99:60:05:EA:88:56:D2:5E:74:9C:47:76:61:27:13:A6:15:BE:67:89:BD:D8:5A:AD:63:F0:08:9E:C2:79:F2\r\na=setup:active\r\na=mid:video\r\na=extmap:2 urn:ietf:params:rtp-hdrext:toffset\r\na=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time\r\na=extmap:4 urn:3gpp:video-orientation\r\na=extmap:5 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01\r\na=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay\r\na=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type\r\na=extmap:8 http://www.webrtc.org/experiments/rtp-hdrext/video-timing\r\na=recvonly\r\na=rtcp-mux\r\na=rtcp-rsize\r\na=rtpmap:100 H264/90000\r\na=rtcp-fb:100 ccm fir\r\na=rtcp-fb:100 nack\r\na=rtcp-fb:100 nack pli\r\na=rtcp-fb:100 goog-remb\r\na=rtcp-fb:100 transport-cc\r\na=fmtp:100 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f\r\na=rtpmap:96 VP8/90000\r\na=rtcp-fb:96 ccm fir\r\na=rtcp-fb:96 nack\r\na=rtcp-fb:96 nack pli\r\na=rtcp-fb:96 goog-remb\r\na=rtcp-fb:96 transport-cc\r\na=rtpmap:97 rtx/90000\r\na=fmtp:97 apt=96\r\na=rtpmap:98 VP9/90000\r\na=rtcp-fb:98 ccm fir\r\na=rtcp-fb:98 nack\r\na=rtcp-fb:98 nack pli\r\na=rtcp-fb:98 goog-remb\r\na=rtcp-fb:98 transport-cc\r\na=rtpmap:99 rtx/90000\r\na=fmtp:99 apt=98\r\na=rtpmap:101 rtx/90000\r\na=fmtp:101 apt=100\r\na=rtpmap:127 red/90000\r\na=rtpmap:124 rtx/90000\r\na=fmtp:124 apt=127\r\na=rtpmap:125 ulpfec/90000\r\nm=application 9 DTLS/SCTP 5000\r\nc=IN IP4 0.0.0.0\r\nb=AS:30\r\na=ice-ufrag:0wiE\r\na=ice-pwd:M5Q8jN2soyvQMce/AA5dUS1V\r\na=ice-options:trickle\r\na=fingerprint:sha-256 B0:99:60:05:EA:88:56:D2:5E:74:9C:47:76:61:27:13:A6:15:BE:67:89:BD:D8:5A:AD:63:F0:08:9E:C2:79:F2\r\na=setup:active\r\na=mid:data\r\na=sctpmap:5000 webrtc-datachannel 1024\r\n"}
Signaling: Headers received [i=409 data(413)]
Conductor: Sending ice candidate.
{"sdpMid":"video","sdpMLineIndex":1,"candidate":"candidate:2600700781 1 tcp 1518280447 172.16.100.228 49936 typ host tcptype passive generation 0 ufrag GQlT network-id 1 network-cost 50"}
Conductor: Sending ice candidate.
{"sdpMid":"data","sdpMLineIndex":2,"candidate":"candidate:2600700781 1 tcp 1518280447 172.16.100.228 49937 typ host tcptype passive generation 0 ufrag GQlT network-id 1 network-cost 50"}
Conductor: Sending ice candidate.
{"sdpMid":"video","sdpMLineIndex":1,"candidate":"candidate:420300814 1 udp 1686052607 210.55.3.198 42702 typ srflx raddr 172.16.100.228 rport 58567 generation 0 ufrag GQlT network-id 1 network-cost 50"}
Conductor: Sending ice candidate.
{"sdpMid":"data","sdpMLineIndex":2,"candidate":"candidate:420300814 1 udp 1686052607 210.55.3.198 34884 typ srflx raddr 172.16.100.228 rport 58568 generation 0 ufrag GQlT network-id 1 network-cost 50"}
Conductor: Sending ice candidate.
{"sdpMid":"audio","sdpMLineIndex":0,"candidate":"candidate:420300814 1 udp 1686052607 210.55.3.198 9795 typ srflx raddr 172.16.100.228 rport 58566 generation 0 ufrag GQlT network-id 1 network-cost 50"}
Signaling: Headers received [i=409 data(413)]
Signaling: Headers received [i=409 data(413)]
Signaling: Headers received [i=422 data(594)]
Conductor: Received candidate : {"sdpMid":"audio","sdpMLineIndex":0,"candidate":"candidate:1951815848 1 udp 2122260223 10.20.30.37 64907 typ host generation 0 ufrag 0wiE network-id 1 network-cost 50"}
Signaling: Headers received [i=409 data(413)]
'AR_Maintenance.exe' (Win32): Loaded 'C:\Windows\System32\Windows.StateRepositoryPS.dll'. Symbols loaded.
Signaling: Headers received [i=422 data(609)]
'AR_Maintenance.exe' (Win32): Loaded 'C:\Windows\System32\capauthz.dll'. Symbols loaded.
Signaling: Headers received [i=409 data(413)]
Conductor: Received candidate : {"sdpMid":"audio","sdpMLineIndex":0,"candidate":"candidate:987079768 1 tcp 1518280447 10.20.30.37 65036 typ host tcptype passive generation 0 ufrag 0wiE network-id 1 network-cost 50"}
The thread 0x155c has exited with code 0 (0x0).
Signaling: Headers received [i=409 data(413)]
Signaling: Headers received [i=409 data(413)]
Signaling: Headers received [i=422 data(627)]
Signaling: Headers received [i=409 data(413)]
'AR_Maintenance.exe' (Win32): Loaded 'C:\Program Files\Intel\Media SDK\mfx_mft_h264ve_32.dll'. Cannot find or open the PDB file.
Conductor: Received candidate : {"sdpMid":"audio","sdpMLineIndex":0,"candidate":"candidate:3708409725 1 udp 1686052607 122.61.107.85 64907 typ srflx raddr 10.20.30.37 rport 64907 generation 0 ufrag 0wiE network-id 1 network-cost 50"}
Signaling: Headers received [i=409 data(413)]
'AR_Maintenance.exe' (Win32): Loaded 'C:\Program Files\Intel\Media SDK\libmfxhw32.dll'. Cannot find or open the PDB file.
The thread 0x1570 has exited with code 0 (0x0).
Channel Open!
DataChannel state in OnOpen: Open
Message Received!
Nice to meet you!
'AR_Maintenance.exe' (Win32): Unloaded 'C:\Program Files\Intel\Media SDK\libmfxhw32.dll'
'AR_Maintenance.exe' (Win32): Loaded 'C:\Program Files\Intel\Media SDK\libmfxhw32.dll'. Cannot find or open the PDB file.
The thread 0xd04 has exited with code 0 (0x0).
The thread 0x108c has exited with code 0 (0x0).
'AR_Maintenance.exe' (Win32): Unloaded 'C:\Program Files\Intel\Media SDK\libmfxhw32.dll'
H264StreamSink::~H264StreamSink()
H264MediaSink::~H264MediaSink()
'AR_Maintenance.exe' (Win32): Loaded 'C:\Program Files\Intel\Media SDK\libmfxhw32.dll'. Cannot find or open the PDB file.
'AR_Maintenance.exe' (Win32): Unloaded 'C:\Program Files\Intel\Media SDK\libmfxhw32.dll'
'AR_Maintenance.exe' (Win32): Loaded 'C:\Program Files\Intel\Media SDK\libmfxhw32.dll'. Cannot find or open the PDB file.
Exception thrown: 'System.TimeoutException' in PeerConnectionClientCore.dll
Timed out long polling, re-trying.
Signaling: Headers received [i=409 data(413)]
The thread 0xc64 has exited with code 0 (0x0).
The thread 0x15ac has exited with code 0 (0x0).
The thread 0x1738 has exited with code 0 (0x0).
Data channel close called and returned
'AR_Maintenance.exe' (Win32): Unloaded 'C:\Program Files\Intel\Media SDK\libmfxhw32.dll'
H264StreamSink::~H264StreamSink()
H264MediaSink::~H264MediaSink()
The thread 0xd60 has exited with code 0 (0x0).
The thread 0xbbc has exited with code 0 (0x0).
The thread 0xeb0 has exited with code 1 (0x1).
The thread 0x620 has exited with code 1 (0x1).
The thread 0x404 has exited with code 0 (0x0).
The thread 0x920 has exited with code 0 (0x0).
The thread 0x6b8 has exited with code 0 (0x0).
The thread 0xf48 has exited with code 0 (0x0).
PeerConnection.Close called and returned
OnReadyToConnect invoked and returned
Garbage collector collects and returns. This is the end of the ClosePeerConnection method
The thread 0x470 has exited with code 0 (0x0).
The thread 0x16a8 has exited with code 0 (0x0).
The thread 0x10e8 has exited with code 0 (0x0).
The thread 0x1668 has exited with code 0 (0x0).
Exception thrown: 'System.TimeoutException' in PeerConnectionClientCore.dll
Timed out long polling, re-trying.
Conductor: Creating peer connection.
Conductor: Getting user media.
The thread 0x6b0 has exited with code 0 (0x0).
VideoEffect Added
Exception thrown at 0x77C9C7D2 (KernelBase.dll) in AR_Maintenance.exe: WinRT originate error - 0xC00DABE4 : 'RecordState'.
Exception thrown at 0x77C9C7D2 in AR_Maintenance.exe: Microsoft C++ exception: Platform::COMException ^ at memory location 0x1F4FFB20. HRESULT:0xC00DABE4 The op is invalid.
WinRT information: RecordState
Stack trace:
 >combase.dll!RoOriginateErrorW(HRESULT error, unsigned int cchMax, const wchar_t * message) Line 563
 >[Inline Frame] Windows.Media.dll!Windows::Media::Report(HRESULT) Line 33
 >Windows.Media.dll!CaptureSharedState::_ValidateRecordStateTransition(RecordState newState) Line 698
 >Windows.Media.dll!CaptureSharedState::TransitionToRecordState(RecordState state) Line 518
 >Windows.Media.dll!Windows::Media::Capture::MediaCaptureServer::StartRecordToCustomSinkAsync(Windows::Media::MediaProperties::IMediaEncodingProfile * encodingProfile, Windows::Media::IMediaExtension * customMediaSink, Windows::Foundation::IAsyncAction * * asyncInfo) Line 4757
 >Org.WebRtc.dll!Windows::Media::Capture::IMediaCapture::StartRecordToCustomSinkAsync(Windows::Media::MediaProperties::MediaEncodingProfile ^ __param0, Windows::Media::IMediaExtension ^ __param1)
 >Org.WebRtc.dll!webrtc::videocapturemodule::CaptureDevice::StartCapture::__l2::Concurrency::task<void> <lambda>(Windows::Media::IMediaExtension ^)::__l2::<lambda>() Line 441
 >[Inline Frame] Org.WebRtc.dll!std::_Invoker_functor::_Call(webrtc::videocapturemodule::CaptureDevice::StartCapture::__l2::Concurrency::task<void> <lambda>(Windows::Media::IMediaExtension ^)::__l2::Concurrency::task<void> <lambda>(void) &)
 >[Inline Frame] Org.WebRtc.dll!std::invoke(webrtc::videocapturemodule::CaptureDevice::StartCapture::__l2::Concurrency::task<void> <lambda>(Windows::Media::IMediaExtension ^)::__l2::Concurrency::task<void> <lambda>(void) &)
 >[Inline Frame] Org.WebRtc.dll!std::_Invoker_ret<Concurrency::task<void>,0>::_Call(webrtc::videocapturemodule::CaptureDevice::StartCapture::__l2::Concurrency::task<void> <lambda>(Windows::Media::IMediaExtension ^)::__l2::Concurrency::task<void> <lambda>(void) &)
 >Org.WebRtc.dll!std::_Func_impl_no_alloc<Concurrency::task<void> <lambda>(void),Concurrency::task<void> >::_Do_call()
 >Org.WebRtc.dll!std::_Func_class<Concurrency::task<void> >::operator()()
 >[Inline Frame] Org.WebRtc.dll!Concurrency::details::_MakeUnitToTFunc::__l2::<lambda_2a153568759a1fa2a7e8447d0218ef52>::operator()(unsigned char) Line 2354
 >[Inline Frame] Org.WebRtc.dll!std::_Invoker_functor::_Call(Concurrency::details::_MakeUnitToTFunc::__l2::<lambda_2a153568759a1fa2a7e8447d0218ef52> &)
 >[Inline Frame] Org.WebRtc.dll!std::invoke(Concurrency::details::_MakeUnitToTFunc::__l2::<lambda_2a153568759a1fa2a7e8447d0218ef52> &)
 >[Inline Frame] Org.WebRtc.dll!std::_Invoker_ret<Concurrency::task<void>,0>::_Call(Concurrency::details::_MakeUnitToTFunc::__l2::<lambda_2a153568759a1fa2a7e8447d0218ef52> &)
 >Org.WebRtc.dll!std::_Func_impl_no_alloc<<lambda_2a153568759a1fa2a7e8447d0218ef52>,Concurrency::task<void>,unsigned char>::_Do_call(unsigned char && <_Args_0>)
 >Org.WebRtc.dll!std::_Func_class<Concurrency::task<void>,unsigned char>::operator()(unsigned char <_Args_0>)
 >Org.WebRtc.dll!Concurrency::task<unsigned char>::_ContinuationTaskHandle<void,void,std::function<Concurrency::task<void> __cdecl(void)>,std::integral_constant<bool,0>,Concurrency::details::_TypeSelectorAsyncTask>::_LogWorkItemAndInvokeUserLambda<std::function<Concurrency::task<void> __cdecl(unsigned char)>,unsigned char>(std::function<Concurrency::task<void> __cdecl(unsigned char)> && _func, unsigned char && _value) Line 3627
 >Org.WebRtc.dll!Concurrency::task<unsigned char>::_ContinuationTaskHandle<void,void,std::function<Concurrency::task<void> __cdecl(void)>,std::integral_constant<bool,0>,Concurrency::details::_TypeSelectorAsyncTask>::_Continue(std::integral_constant<bool,0> __formal, Concurrency::details::_TypeSelectorAsyncOperationOrTask __formal) Line 3674
 >[Inline Frame] Org.WebRtc.dll!Concurrency::task<unsigned char>::_ContinuationTaskHandle<void,void,std::function<Concurrency::task<void> __cdecl(void)>,std::integral_constant<bool,0>,Concurrency::details::_TypeSelectorAsyncTask>::_Perform() Line 3632
 >Org.WebRtc.dll!Concurrency::details::_PPLTaskHandle<unsigned char,Concurrency::task<unsigned char>::_ContinuationTaskHandle<void,void,std::function<Concurrency::task<void> __cdecl(void)>,std::integral_constant<bool,0>,Concurrency::details::_TypeSelectorAsyncTask>,Concurrency::details::_ContinuationTaskHandleBase>::invoke() Line 1473
 >Org.WebRtc.dll!Concurrency::details::_TaskProcHandle::_RunChoreBridge(void * _Parameter) Line 160
 >Org.WebRtc.dll!Concurrency::details::_DefaultPPLTaskScheduler::_PPLTaskChore::_Callback(void * _Args) Line 52
 >msvcp140_app.dll!Concurrency::details::`anonymous namespace'::_Task_scheduler_callback(_TP_CALLBACK_INSTANCE * _Pci, void * _Args, _TP_WORK * __formal) Line 158
 >ntdll.dll!TppWorkpExecuteCallback()
 >ntdll.dll!TppWorkerThread()
 >ntdll.dll!__RtlUserThreadStart()
 >ntdll.dll!__RtlUserThreadStart@8()
'AR_Maintenance.exe' (Win32): Unloaded 'C:\Windows\System32\deviceaccess.dll'
Unhandled exception at 0x774A273B (ucrtbase.dll) in AR_Maintenance.exe: Fatal program exit requested.
```

**Comment by Fritzwald** [comment]
When you end a call, does the camera stay on? (light on front of hololens + red camera icon top left corner)

**Comment by jamescadd** [comment]
I need to update my device and will give it another try.  One thing I've heard in the past is that other apps that access the camera can cause issues, for example running a Skype call then trying the sample.  Hard to tell from the logs above what's going on but it sounds like it is easy to repro.  I'll let you know when I try this (likely Monday).

**Comment by morosev** [comment]
I agree about previous comment. It looks like the camera session from the previous call is not close down correctly.

**Comment by jamescadd** [comment]
Small update to let you know I'm having some issues with VS after moving from the 2018 Unity tools which we're using with the latest updates back to the 2017 tools.  This may take a bit longer to repro as I get a system to do the build in order.

**Comment by Jelop** [comment]
Hey guys, my apologies for the late reply. I have managed to solve the issue. When I first set up my project I had added the block of xml from the PeerCC-Sample readme. But in the process of development I had to delete the unity generated app folder and completely forgot to re-modify Package.appxmanifest.  I saw it mentioned in #41 today and so I checked the file, modified it, and since then have been unable to reproduce the problem. Apologies for any time wasted.
